packages/opds-web-client/src/components/BookDetails.tsx

```diff
@@ -19,6 +19,11 @@ export default class BookDetails<P extends BookDetailsProps> extends Book<P> {
           </div>
           <div className="header">
             <h1 className="title">{this.props.book.title}</h1>
+            {
+              this.props.book.subtitle ?
+              <h3 className="subtitle">{ this.props.book.subtitle }</h3> :
```

Comment:
I see that the current heading structure is
```
H1 // title
H3 // if there is a subtitle
H2 // authors
```

It'd be better if the H2 came before the H3. I'm not sure this should be an h3 or authors an h2 but not part of the PR (and I think technically only the homepage of a site should have an h1).

Reply:
Hmm, I think authors are more important than subtitle, but it seems like it would look weird to put the subtitle below the authors.

Reply:
This is also a problem with series.

Reply:
I agree it would look weird. This should be a separate conversation.

Reply:
Hey, this is Risa - would you like me to bring this to our accessibility/UX team for recommendations, or discuss on your own?

Reply:
Yeah I think that would be useful.

Reply:
Yes, @killerpuppytails can you make a ticket so Willa and I can work on it? I'll then loop you in @aslagle .

Reply:
Sure thing!

Reply:
SIMPLY-2022 created and assigned to you @EdwinGuzman